### ISSUE TITLE
Option to force // MARK styling

### ISSUE THREAD

**Issue by kevinrenskers** [nicklockwood/SwiftFormat#196]
In our code, in which multiple developers are working, we have two styles of adding the MARK comment: surrounded by newlines top and bottom, and without the newline at the bottom.

```
override func viewDidLoad() {
 ...
}

// MARK: - UITableViewDataSource

func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
  ...
}
```

versus

```
override func viewDidLoad() {
 ...
}

// MARK: - UITableViewDataSource
func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
  ...
}
```

It would be great to force a similar style in our code, possibly configurable.

**Comment by kevinrenskers** [comment]
(Personally I'd say that the default should have newlines above and below, since usually this MARK is not about one method but a group of them, so having it as a method comment is confusing.)

**Comment by nicklockwood** [comment]
Implemented in 0.32.0

**Comment by sisoje** [comment]
> Implemented in 0.32.0

please how?

**Comment by LeaneTEXIER** [comment]
Hello,
The markTypes and organizeDeclarations rules add a line before and after even if we disable blankLinesAroundMark rule. Can we change that ? Is it possible to add only the line before the mark and no the one after ?
